source/nordic-sdk/components/softdevice/s130/headers/nrf_svc.h

```diff
@@ -49,7 +49,7 @@
   { \
     __asm( \
         "svc %0\n" \
-        "bx r14" : : "I" ((uint32_t)number) : "r0" \
```

Comment:
@rgrover : they are not new. These are the same warnings generated using the previous version of the sdk. I suppose what I am asking is how feasible is to modify these files to fix this if they are meant to be easily replaceable...